libs/datasets/combined_datasets.py

```diff
@@ -138,6 +141,69 @@ def get_us_latest_for_fips(fips) -> dict:
     return us_latest.get_record_for_fips(fips)
 
 
+def _remove_padded_nans(df, columns):
+    if df[columns].isna().all().all():
+        return df.loc[[False] * len(df), :]
+
+    first_valid_index = min(df[column].first_valid_index() for column in columns)
+    last_valid_index = max(df[column].last_valid_index() for column in columns)
+    df = df.iloc[first_valid_index:last_valid_index]
+    return df.reset_index(drop=True)
+
+
+def get_timeseries_for_fips(
+    fips: str, columns: List = None, remove_padding_nans: bool = False
```

Comment:
does columns need to have a default? refactoring later will likely be easier if callers always explicitly list the columns requested.

Reply:
hmm I guess that's true - I guess in terms of how this function is used I wouldn't expect to have to pass the columns.  I primarily see it as a wrapper over getting the entire dataset with a bit more optionality.